product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/index.mdx

```diff
@@ -40,3 +45,34 @@ edb_admin=> \conninfo
 You are connected to database "edb_admin" as user "edb_admin" on host "xxxxxxxxx.xxxxx.edbcloud.io" at port "5432".
 SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
 ```
+
+## Setting up Azure Infrastructure to Connect to a Private Network Cluster
+
+The Private Networking option offers a higher level of isolation and security to your EDB Cloud cluster. Clusters with Private Networking enabled are not accessible from outside of your EDB Cloud resource virtual network. Only resources from the same virtual network can access the private EDB Cloud cluster. 
+
+!!! Note
+    EDB strongly discourages users from provisioning additional resources in the EDB Cloud resource virtual network.
+
+You can connect to the private EDB Cloud cluster either from an application in Azure 
+or from an on-premises network. The following sections contain walkthrough examples to guide you through each of the options:
+
+- [Connecting from Azure](01_connecting_from_azure)
+- [Connecting from an On-Premise Network](02_connect_on_premises)
+
+### Connection Walkthrough Examples
```

Comment:
As noted elsewhere, I think this should be embedded in the examples to help customers understand them.

Reply:
Done. Moved the prerequisites to the start of each of the examples.